release-notes.txt

```diff
@@ -0,0 +1,19 @@
+mcuboot 0.9 - Release Notes
```

Comment:
Do we want to put this document in either markdown or RST. This formatting looks like RST, and naming it that would make it format nicely when viewed in github.